ui/raidboss/data/05-shb/raid/e8n.txt

```diff
@@ -4,4 +4,105 @@
 hideall "--Reset--"
 hideall "--sync--"
 
+# -ii 4D58 4DD3 4DDF 4DE0 4DF3 4DF5 4E0B 4E0D 4E10 4E11 4E18 4FCA
+
 0.0 "--Reset--" sync / 21:........:40000010:/ window 10000 jump 0
+
+# Opening block
+0.0 "--sync--" sync /:Engage!/ window 0,1
+1.4 "--sync--" sync /:Shiva:4DD3:/ window 1.4,2
+15.1 "Absolute Zero" sync /:Shiva:4DD7:/ window 15,5
+27.2 "Redress" sync /:Shiva:4F4F:/
+32.4 "Shining Armor" sync /:Shiva:4DF1:/
+40.4 "Axe Kick/Scythe Kick" sync /:Shiva:4DE[23]:/
+52.5 "Redress" sync /:Shiva:4F4E:/
+57.6 "Frost Armor" sync /:Shiva:4DF0:/
+65.6 "Biting Frost/Driving Frost" sync /:Shiva:4DD[BC]:/
+78.8 "Double Slap" sync /:Shiva:4DDA:/
+92.0 "Diamond Frost" sync /:Shiva:4DE1:/
+100.2 "Frigid Water/Frigid Stone" # sync /:Shiva:4E(?:08|66):/
```

Comment:
I know this is nonsensical, but can you remove all the `?:` in the regex groups? This breaks `python util/test_timeline.py` (although maybe that could just get fixed too).

Reply:
Sure. I've been temporarily been stripping them for testing the timeline, but I suppose the overhead is negligible enough to just accept the loss from capturing.